docs/EN_US/Installing_and_RunningTheHPCCPlatform/Installing_and_RunningTheHPCCPlatform.xml

```diff
@@ -2434,20 +2434,10 @@ add1(10);
         <title>Python</title>
 
         <para>The HPCC Systems platform supports Python3 by default. It
-        includes plugins for both Python2 and Python3, but only one may be
+        includes a plug-in for Python3, and only one version of Python may be
         safely enabled at a time as the Python libraries export the same
-        symbols for both versions. Enabling both may lead to unpredictable
-        results including segmentation faults or undefined symbol
-        errors.</para>
-
-        <para>By default the Python2 plugin is present but disabled, and the
-        Python3 plugin is present and enabled. If you want to use Python2
-        modify your environment.conf file according to the example in the
-        file.</para>
-
-        <para>A cluster updated from an older version may not have the
-        appropriate entries in the environment.conf file to support either
-        version of Python.</para>
+        symbols for different versions. The Python3 plug-in is present and
+        enabled while Python2 is no longer supported.</para>
```

Comment:
The HPCC Systems platform supports Python3 and the Python3 plug-in is present and
        enabled. Python2 is no longer supported.</para>